src/meta_schedule/extracted_task.cc

```diff
@@ -32,6 +38,42 @@ ExtractedTask::ExtractedTask(String task_name, IRModule mod, Target target,
   data_ = n;
 }
 
+Optional<tir::PrimFunc> DefaultTaskFilter(const Array<te::Tensor>& args) {
+  using namespace ::tvm::te;
+  std::vector<Tensor> stack;
+  std::unordered_set<const TensorNode*> visited;
+  for (const Tensor& v : args) {
+    for (const PrimExpr& e : v->shape) {
+      // Dynamic shape is not supported for now
+      if (!e->IsInstance<IntImmNode>()) {
+        return NullOpt;
+      }
+    }
+    if (!visited.count(v.get())) {
+      visited.insert(v.get());
+      stack.push_back(v);
+    }
+  }
+  while (!stack.empty()) {
+    Tensor tensor = stack.back();
+    stack.pop_back();
+    if (tensor->op->IsInstance<PlaceholderOpNode>()) {
+      // do nothing
+    } else if (tensor->op->IsInstance<ComputeOpNode>()) {
```

Comment:
```suggestion
    } else if (tensor->op->IsInstance<ComputeOpNode>() || tensor->op->IsInstance<ExternOpNode>()) {
```

You'll need to check for extern op's here as well ([ref](https://github.com/apache/tvm/pull/11692/files#diff-a3afd47a39285ff4100ee476114ca7a33f06b5a0929d8ecf6623c53c0de01ad8L55)).

Reply:
This is certain an interesting design tradeoff. MetaSchedule/AutoScheduler default rules are not supposed to handle all the ExternOp and HybridOp in every model, because they are already scheduled (e.g. the NMS operator), or do not need to be scheduled (e.g. external library). Therefore, we are filtering out all those operations by default to make sure MetaSchedule doesn't break on customer models.

Given we support customized filtering methods, do you think it's a good idea if we expose another method that doesn't rule out extern-op for customized usecases?